### ISSUE TITLE
[AXE-CORE]: Page must contain a level-one heading

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#5947]
## Description

When a user logs into an account that has no ratings, an error is shown. However, there is no `<h1>` on the page, which creates a confusing experience for screen reader users.

## Point of Contact

**VFS Point of Contact:**  Jennifer

## Acceptance Criteria

As a screen reader user, I want to navigate a page by headings, starting with an `<h1>`.

## Steps to Recreate

1. Enter `https://staging.va.gov/disability/view-disability-rating/rating` in browser
2. Log in as vets.gov.user+10
3. Run axe scan
4. Verify that an h1 is missing

## Possible Fixes (optional)

Add an `<h1>` for "Your VA disability rating" and use CSS to suppress it for sighted users.

## WCAG or Vendor Guidance (optional)

* [Page must contain a level-one heading](https://dequeuniversity.com/rules/axe/3.4/page-has-heading-one?application=AxeChrome)
* [w3c Headings](https://www.w3.org/WAI/tutorials/page-structure/headings/)

**Comment by jason-gcio** [comment]
Strange, thought for sure there was an `<h1>` on this page but now I see it has another type of error message.
![h1-staging](https://user-images.githubusercontent.com/53826255/74766560-50d32700-5253-11ea-8d9e-4cf22238588b.JPG)

**Comment by jason-gcio** [comment]
Quick chat with the team and someone mentioned that this might be an LOA1 user who has no info.  Still seeing how we can address it with a title, so we will triage this with the rest.

**Comment by jenstrickland** [comment]
@jason-gcio I think this issue has been addressed, since I just checked and there's an h1.
